src/recovery/impl/thread_local_logger.h

```diff
@@ -38,16 +38,19 @@ namespace Recovery {
 
 class ThreadLocalLogger final : public LoggerBase {
  public:
-  ThreadLocalLogger();
+  ThreadLocalLogger(const Config&);
   void RememberMe(const EpochNumber) final override;
   void Enqueue(const WriteSetType& ws_ref_, EpochNumber epoch,
                bool entrusting) final override;
   void FlushLogs(EpochNumber stable_epoch) final override;
   void TruncateLogs(
       const EpochNumber checkpoint_completed_epoch) final override;
   EpochNumber GetMinDurableEpochForAllThreads() final override;
+  std::string GetLogFileName(size_t thread_id);
+  std::string GetWorkingLogFileName(size_t thread_id);
```

Comment:
I recommend to add `const` after function declarations to declare that "these functions are const-functions", which ensure no side effects.
e.g., 

```
  std::string GetLogFileName(size_t thread_id) const;
  std::string GetWorkingLogFileName(size_t thread_id) const;
```